dbaas/dbaas.go

```diff
@@ -123,6 +123,19 @@ func (p Cmd) apply(k8sObj string) error {
 	return nil
 }
 
+func (p Cmd) Annotate(resource, clusterName, annotName, instance string) error {
+	_, err := p.runCmd(execCommand, "annotate", resource, clusterName, annotName+"="+instance, "--overwrite=true")
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (p Cmd) GetSecret(secretName string) ([]byte, error) {
+	return p.runCmd(execCommand, "get", "secret", secretName, "-o", "json")
+}
+
```

Comment:
No need in this method. There is a `GetObject` method which might be used instead.